recipes/micromagneticdata/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "micromagneticdata" %}
+{% set version = "0.12" %}
+{% set sha256 = "010a38a63ff3ca6db32399d45c94c5b66591afac4c49ff9b3d4fb959e1f30b80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - discretisedfield
+    - oommfodt
+    - ipywidgets
+    - pytest
+
+test:
+  imports:
+    - micromagneticdata
+    - micromagneticdata.tests
+
+  commands:
+    - python -c "import sys; import micromagneticdata as m; sys.exit(m.test())"
+
+
+about:
+  home: http://joommf.github.io/
+  license: BSD 3-clause
+  summary: Python tools for analysing micromagnetic data in a Jupyter notebook.
+  license_family: BSD
+  license_file: LICENSE
+
+  description: |
+    micromagneticdata is a Python package that provides the analysis
+    of micromagnetic data. It is available on all major operating
+    systems (Windows, MacOS, Linux) and requires Python 3.6 or higher.
+
+  doc_url: http://micromagneticdata.readthedocs.io
+  dev_url: https://github.com/joommf/micromagneticdata
+
+extra:
+  recipe-maintainers:
+    - marijanbeg
+    - fangohr
+    - sergii-mamedov
+    - takluyver
+    - rpep
```

Comment:
All of the maintainers here need to confirm they are ok with this.

Reply:
@fangohr, @sergii-mamedov, @takluyver, @rpep Do you all agree to be the maintainers of the package? If yes, please confirm.

Reply:
yes

Reply:
yes

Reply:
yes